src/celeritas/mat/MaterialParams.cc

```diff
@@ -89,18 +111,16 @@ MaterialParams::from_import(ImportData const& data)
 
         for (auto const& elem_comp : material.elements)
         {
-            ElementId elem_def_id{elem_comp.element_id};
-
             // Populate MaterialParams number fractions
             material_params.elements_fractions.push_back(
-                {elem_def_id, elem_comp.number_fraction});
+                {static_cast<ElementId>(elem_comp.element_id),
```

Comment:
You shouldn't be `static_cast`ing an `int` to an `Id`; instead, cast to `unsigned int` and let the implicit conversion work. But if you take the recommendation above and make `element_id` an unsigned int, then you can just write 
```suggestion
                {elem_comp.element_id,
```